build.proj

```diff
@@ -36,23 +36,21 @@
           DependsOnTargets="$(TraversalBuildDependencies)" />
 
   <Target Name="BuildCustomTasks">
-      <MSBuild Projects="tools-local/Microsoft.DotNet.Build.Tasks.Local/Microsoft.DotNet.Build.Tasks.Local.builds" />
       <MSBuild Projects="tools-local/tasks/core-setup.tasks.builds" />
 
     <PropertyGroup>
       <DependencyModelTFM>netstandard1.3</DependencyModelTFM>
       <DependencyModelTFM Condition="'$(MSBuildRuntimeType)' != 'Core'">net451</DependencyModelTFM>
     </PropertyGroup>
     
-      <ItemGroup>
-        <CustomTaskDependencies Include="$(BuildToolsTaskDir)Newtonsoft.Json.dll" />
-        <CustomTaskDependencies Include="$(BuildToolsTaskDir)Microsoft.DotNet.PlatformAbstractions.dll" />
```

Comment:
I can add this back.  This was originally added when this repo was using this task, https://github.com/dotnet/core-setup/blob/master/tools-local/Microsoft.DotNet.Build.Tasks.Local/GetHostInformation.cs.  Now that we're using the task from BuildTools, it appeared safe to remove this, and local builds on Linux / Windows showed it was not necessary.  There are no other mechanisms copying this assembly there that I am aware of, so I'm not certain why the tasks in core-setup.tasks were not complaining about the missing dependency.